frontend/test/__support__/e2e/cypress.js

```diff
@@ -31,3 +32,15 @@ export * from "./helpers/e2e-embedding-helpers";
 export * from "./helpers/e2e-permissions-helpers";
 
 Cypress.on("uncaught:exception", (err, runnable) => false);
+
+const titleToFileName = title => title.replace(/[>]/g, "");
```

Comment:
This will replace the `>` for `space`, the way that the file is saved

<img width="1286" alt="Screen Shot 2022-05-06 at 12 22 58" src="https://user-images.githubusercontent.com/17742979/167190469-9dd35a42-af84-4dcb-bf01-f582c53caa7f.png">